src/bloggy_frontend/src/utils/checkResult.jsx

```diff
@@ -1,9 +1,10 @@
 export const checkResult = (result) => {
-  if (result.hasOwnProperty('ok')) {
-    return { success: true, data: result.ok }
-  } else if (result.hasOwnProperty('err')) {
-    return { success: false, error: result.err }
-  } else {
+  if (!result.hasOwnProperty('ok')) {
+    if (result.hasOwnProperty('err')) {
+      return { success: false, error: result.err }
+    }
     throw new Error('Unexpected result format')
```

Comment:
Is this throw required here?

Reply:
The throw statement in the code serves as a safeguard to handle unexpected cases where the result object does not contain either the ok or err property.  By throwing an error, it ensures that any unexpected format of the result object is caught and handled appropriately.

Reply:
ok then you can keep it